resources/operator.yaml

```diff
@@ -34,14 +34,14 @@ spec:
           - /usr/local/bin/ao-logs
           - /tmp/ansible-operator/runner
           - stdout
-          image: quay.io/benchmark-operator/benchmark-operator:master
+          image: quay.io/hnhan/benchmark-operator:uperf-scale
```

Comment:
Need to leave this as our operator image.

Reply:
Will chnage this and the other images back to official